src/main/java/io/vinyldns/java/model/batch/CreateBatchRequest.java

```diff
@@ -21,6 +21,8 @@
 
   private List<ChangeInput> changes;
 
+private String ownerGroupId;
+
   public CreateBatchRequest(String comments, List<ChangeInput> changes) {
```

Comment:
`ownerGroupId` can be added here as well (and the other constructor should pass in `null` for the value).